src/main/java/io/reactivex/Single.java

```diff
@@ -2173,6 +2173,9 @@
      * is executed once per subscription.
      * <p>Note that the {@code onFinally} action is shared between subscriptions and as such
      * should be thread-safe.
+     * <p>
+     * <img width="1040" height="472" src="https://user-images.githubusercontent.com/4089934/39448132-dca93836-4cbb-11e8-9b13-ce078a8ba206.png" alt="">
```

Comment:
Images should be 640 width and the height should be reduced to keep the aspect ratio. In this case, 2079x945 -> 640x290